internal/pkg/state/state.go

```diff
@@ -46,51 +47,47 @@ type State struct {
 }
 
 type Options struct {
-	fs                filesystem.Fs
-	manifest          *manifest.Manifest
-	api               *remote.StorageApi
-	schedulerApi      *scheduler.Api
-	context           context.Context
-	logger            log.Logger
 	LoadLocalState    bool
 	LoadRemoteState   bool
 	IgnoreNotFoundErr bool // not found error will be ignored
 }
 
-func NewOptions(projectDir filesystem.Fs, m *manifest.Manifest, api *remote.StorageApi, schedulerApi *scheduler.Api, ctx context.Context, logger log.Logger) *Options {
-	return &Options{
-		fs:           projectDir,
-		manifest:     m,
-		api:          api,
-		schedulerApi: schedulerApi,
-		context:      ctx,
-		logger:       logger,
-	}
+// ObjectsContainer is Project or Template.
+type ObjectsContainer interface {
```

Comment:
Spolocny interface pre project aj template.

Reply:
Je to potrebne, pretoze project/template sa nacitavaju rozne, preto to nemoze byt v `dependecies`, ako novy interface/parameter.